src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -537,89 +596,93 @@ void func_80B14E28(EnTa* this, PlayState* play) {
     Play_CameraSetAtEye(play, this->subCamId, &subCamAt, &subCamEye);
 }
 
-void func_80B14EDC(EnTa* this, PlayState* play) {
+void EnTa_RemoveFloorCamera(EnTa* this, PlayState* play) {
     Play_ChangeCameraStatus(play, this->returnToCamId, CAM_STAT_ACTIVE);
     Play_ClearCamera(play, this->subCamId);
 }
 
-void func_80B14F20(EnTa* this, EnTaActionFunc arg1) {
-    func_80B13AA0(this, arg1, func_80B16854);
-    this->eyeIndex = 2;
+void EnTa_SetActionFuncWithBackToSleepAnimation(EnTa* this, EnTaActionFunc actionFunc) {
+    EnTa_SetActionFunctions(this, actionFunc, EnTa_AnimSitSleeping);
+    this->eyeIndex = TALON_EYE_INDEX_CLOSED;
     Animation_Change(&this->skelAnime, &gTalonSitSleepingAnim, 1.0f, 0.0f,
                      Animation_GetLastFrame(&gTalonSitSleepingAnim), ANIMMODE_ONCE, -5.0f);
-    this->unk_2E2 = 0;
+    this->nodOffTimer = 0;
     this->currentAnimation = &gTalonSitSleepingAnim;
 }
 
-void func_80B14FAC(EnTa* this, EnTaActionFunc arg1) {
-    this->eyeIndex = 1;
-    func_80B13AA0(this, arg1, func_80B16938);
-    this->unk_2E0 &= ~0x10;
+void EnTa_SetActionFuncWithWakeUpAnimation(EnTa* this, EnTaActionFunc actionFunc) {
+    this->eyeIndex = TALON_EYE_INDEX_HALF;
+    EnTa_SetActionFunctions(this, actionFunc, EnTa_AnimRunToEnd);
+    this->stateFlags &= ~TALON_STATE_FLAG_ANIMATION_FINISHED;
     Animation_Change(&this->skelAnime, &gTalonSitWakeUpAnim, 1.0f, 0.0f, Animation_GetLastFrame(&gTalonSitWakeUpAnim),
                      ANIMMODE_ONCE, -5.0f);
 }
 
-void func_80B15034(EnTa* this, PlayState* play) {
+void EnTa_TalkNotEnoughRupees(EnTa* this, PlayState* play) {
     if ((Message_GetState(&play->msgCtx) == TEXT_STATE_EVENT) && Message_ShouldAdvance(play)) {
         Message_CloseTextbox(play);
-        func_80B14F20(this, func_80B16504);
-        func_80B13AAC(this, play);
+        EnTa_SetActionFuncWithBackToSleepAnimation(this, EnTa_IdleSittingInLonLonHouse);
+        EnTa_SetTextForTalkInLonLonHouse(this, play);
     }
-    this->unk_2E0 |= 1;
+    this->stateFlags |= TALON_STATE_FLAG_TRACKING_PLAYER;
 }
 
-s32 func_80B150AC(EnTa* this, PlayState* play, s32 idx) {
+bool EnTa_IsPlayerHoldingSuperCucco(EnTa* this, PlayState* play, s32 cuccoIdx) {
```

Comment:
We don't use `bool`, because many booleans are typed by the game as not-int, so for consistency reasons we just don't use it at all